seatunnel-engine/seatunnel-engine-server/src/main/java/org/apache/seatunnel/engine/server/dag/physical/PhysicalPlanGenerator.java

```diff
@@ -274,15 +274,14 @@ private List<PhysicalVertex> getCommitterTask(
                             }
                             // if sinkAggregatedCommitter is empty, don't create task.
                             if (sinkAggregatedCommitter.isPresent()) {
-                                long taskGroupID = idGenerator.getNextId();
-                                long taskTypeId = idGenerator.getNextId();
+                                long taskGroupID = taskGroupIdGenerator.getNextId();
                                 TaskGroupLocation taskGroupLocation =
                                         new TaskGroupLocation(
                                                 jobImmutableInformation.getJobId(),
                                                 pipelineIndex,
                                                 taskGroupID);
                                 TaskLocation taskLocation =
-                                        new TaskLocation(taskGroupLocation, taskTypeId, 0);
+                                        new TaskLocation(taskGroupLocation, 1, 0);
```

Comment:
It's a little werid some index start with 0 but some index start with 1. Let's all start with 0.

Reply:
Done